crates/sui/src/keytool.rs

```diff
@@ -551,9 +553,16 @@ impl KeyToolCommand {
                 let keys = keystore
                     .keys()
                     .into_iter()
-                    .map(Key::from)
-                    .collect::<Vec<_>>();
-
+                    .map(|pk| {
+                        let mut key = Key::from(pk);
+                        let alias = keystore.get_alias_by_address(&key.sui_address);
+                        alias.map(|x| {
+                            key.alias = Some(x);
+                            key
+                        })
```

Comment:
Thanks for suggesting this. `get_alias_by_address` returns a `Result`, thus that is why I am using a map on `alias` as this collection returns a `<Result<Vec<_>, _>>()`, so that I can call `map_err` on it when there's an error. Making this assignment `key.alias = alias;` wouldn't work, unless I'm missing something.

Reply:
Man, 0 for 2 on noticing `Result` types in PRs, for me... a `map` call that is expressly to mutate a variable seems a little strange to me, still though.

Looking at the implementation of `get_alias_by_address`, it looks like it returns an error when you want to set `key.alias` to `None`, and `Ok(alias)` otherwise.  In which case you can use `Option::ok` to do the conversion:

```rust
key.alias = alias.ok();
```

Reply:
I think here it's fine, because by design it should always have an alias for that address.